src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +158,27 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
```

Comment:
just ensure modifiers are in a pre-defined List {void,public,private etc} ..

Reply:
Agreed, added to the constant file and implemented a check to verify modifiers being sent in are all supported.